src/api/general-services/pipeline-manage/index.js

```diff
@@ -180,8 +180,8 @@ const buildStateMachineDefinition = (skeleton, context) => {
 };
 
 const createQCPipeline = async (experimentId, processingConfigUpdates, authJWT) => {
-  const accountId = await config.awsAccountIdPromise;
-  const roleArn = `arn:aws:iam::${accountId}:role/state-machine-role-${config.clusterEnv}`;
+  const accountId = config.awsAccountId;
+  const roleArn = `arn:aws:iam::${config.awsAccountId}:role/state-machine-role-${config.clusterEnv}`;
```

Comment:
Line 183 can be removed, or line 184 changed to use `accountId` instead of `config.awsAccountId`

Reply:
Line 266-267 should use a pattern same to the one implemented here for consistency.